src/components/InputNumber.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div class="inputnumber">
+    <Input v-model="inputValue" />
+    <font-awesome-icon @click="increase" class="increase" icon="angle-up" />
+    <div class="unit">
+      <p class="text">WIT</p>
+    </div>
+  </div>
+</template>
+
+<script>
+import Input from './Input'
+
+export default {
+  name: 'InputNumber',
+  components: {
+    Input,
+  },
+  props: {
+    value: [String, Number],
+  },
+  computed: {
+    inputValue: {
+      get() {
+        return this.value
+      },
+      set(inputValue) {
+        this.$emit('input', inputValue)
+      }
+    }
+  },
+  methods: {
+    increase() {
+      this.$emit('input', parseInt(this.value) + 1)
```

Comment:
Just to be safe

```suggestion
      this.$emit('input', parseInt(this.value, 10) + 1)
```

Reply:
Do you think that the expected behavior is to change the base of the introduced number? IMHO we should only accept numbers

Reply:
Entering a monetary value in hexadecimal should be a hard error.